src/main/java/com/github/pascalgn/jiracli/Jiracli.java

```diff
@@ -100,6 +106,15 @@ public static void main(String[] args) {
         map.put(Option.GUI, list.contains("-g") || list.contains("--gui"));
         map.put(Option.VERSION, list.contains("-V") || list.contains("--version"));
         list.removeAll(Arrays.asList("-h", "--help", "-c", "--console", "-g", "--gui", "-V", "--version"));
+        String line = null;
+        int commandIndex = list.indexOf("--command") + 1;
+        if (commandIndex > 0 && (list.size() > commandIndex) ) {
+            line = list.get(commandIndex);
```

Comment:
You could use `list.remove` instead of list.get, then you don't need to remove the line later

Reply:
Have used remove for getting the command line and added the --command option to the other removeAll.